Makefile

```diff
@@ -47,17 +31,5 @@ update:
 
 release:
 	@$(CARGO) build --release
-	wascap sign $(RELEASE)/{{crate_name}}.wasm $(RELEASE)/{{crate_name}}_signed.wasm -i $(KEYDIR)/account.nk -u $(KEYDIR)/module.nk -s -n {{crate_name}}
-	
-keys: keys-account
-keys: keys-module
-
-keys-account:
-	@mkdir -p $(KEYDIR)
-	nk gen account > $(KEYDIR)/account.txt
-	awk '/Seed/{ print $$2 }' $(KEYDIR)/account.txt > $(KEYDIR)/account.nk
-
-keys-module:
-	@mkdir -p $(KEYDIR)
-	nk gen module > $(KEYDIR)/module.txt
-	awk '/Seed/{ print $$2 }' $(KEYDIR)/module.txt > $(KEYDIR)/module.nk
+	VERSION=$(cargo metadata --no-deps --format-version 1 | jq -r '.packages[].version')
+    wash claims sign $(RELEASE)/{{crate_name}}.wasm -c wasmcloud:httpserver --name "{{crate_name}}" --ver $VERSION --rev 0
```

Comment:
Make sure this doesn't have spaces, `wash` needs to be preceded by a tab.

Also, if you decide to move the `VERSION` variable to the top-level section, change `$VERSION` to `$(VERSION)`